README.md

```diff
@@ -2,7 +2,26 @@
 
 This repo contains the exploratory notebooks, modelling and analysis scripts to find out what tech Wellcome funds, and also the code to create the data representation labels.
 
-## Set-up
+### Contents
+1. [Set-up](#setup)
+2. [Tech Wellcome Funds](#tech)
+  - [Pipeline overview](#pipelineoverview)
+    - [Create the training data](#pipeline1)
```

Comment:
just a note on terminology pipeline usually refers to all steps, I think those individual scripts are steps of the pipeline. when I mentioned to record your pipelines I mostly meant to give an overview of the different whole pipelines that someone needs to run to recreate your results. for example one pipeline can be 1) create data 2) train model 3) evaluate model

note that some tests may be common in more than one pipelines. it is worth thinking in terms of a DAG (something that data science cookie cutter also mentions) and DVC enforces. See https://dvc.org/doc/command-reference/dag for a visual that might help

Reply:
Maybe me labelling the section as "#pipeline1" for the contents has made this confusing, I don't mean it is the first pipeline, but 'pipeline step 1'. There is only one pipeline, similar to your example it consists of the 5 subsubsections starting with step 1 'Create the training data'. Or have I misunderstood something?